### PR TITLE
feat(wp) : added support of cashaddr for create address using SDK

### PR DESCRIPTION
**Pull request by mukeshsp**
Jira Ticket : https://bitgoinc.atlassian.net/browse/BG-35555

### CODE REVIEW

File: modules/core/src/v2/wallet.ts [hunk 1]
```diff
@@ -264,6 +264,7 @@ export interface CreateAddressOptions {
   label?: string;
   lowPriority?: boolean;
   forwarderVersion?: number;
+  format?: string;
```
Comment (tylerlevine) [inline modules/core/src/v2/wallet.ts:267]:
```suggestion
  format?: 'base58' | 'cashaddr';
```

**Review by tylerlevine — COMMENTED** [review]
looks good to me, just a slightly tighter type on the new `format` property would be nice

**Review by tylerlevine — CHANGES_REQUESTED** [review]
forgot to request changes, see inline comment

File: modules/core/src/v2/wallet.ts (hunk 1 above)
Comment (mukeshsp) [reply modules/core/src/v2/wallet.ts]: updated the changes as suggested.

**Review by tylerlevine — APPROVED** [review]

**Pull request merged**